format/spec.md

```diff
@@ -736,8 +738,6 @@ A snapshot's `first-row-id` is assigned to the table's current `next-row-id` on
 
 The snapshot's `first-row-id` is the starting `first_row_id` assigned to manifests in the snapshot's manifest list.
 
-The snapshot's `added-rows` is the sum of all the  [`added_rows_count`](#manifest-lists) in all added manifests.
```

Comment:
@RussellSpitzer, I've removed `added-rows` because I think that it is misleading and makes row ID range assignment more complicated.

In this PR, row ID range assignment is based on the total number of existing or added rows in new manifests. That leaves room for any data files that are missing `first_row_id` even if the files are `existing` and not `added`. That may not be the total number of added rows so I don't think it makes sense to have `added-rows` in the snapshot.

Reply:
That's fine, we only had this added in order to pass the information from the snapshot into the table metadata. I believe now that logic has moved we don't have that issue.